crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -1153,6 +1153,32 @@ TEST_P(ECDH_ServiceIndicatorTest, ECDH) {
   CALL_SERVICE_AND_CHECK_APPROVED(approved, ASSERT_TRUE(ECDH_compute_key_fips(digest.data(),
                               digest.size(), EC_KEY_get0_public_key(peer_key.get()), our_key.get())));
   ASSERT_EQ(approved, ecdhTestVector.expect_approved);
+
```

Comment:
There is a typo up there in ECDSAKeyCheck on l. 964, where it says:
// Test running EVP_PKEY_derive
which I think is supposed to be "keygen"

Reply:
Done.